tests/integration/helpers.bash

```diff
@@ -396,7 +402,7 @@ function requires() {
 			fi
 			;;
 		root)
-			if [ $EUID -ne 0 ]; then
+			if [ $EUID -ne 0 ] || in_userns; then
```

Comment:
Hmm, this change probably results in many tests being skipped which were not skipped before.

Reply:
We don't run the suite in user namespaces, though? But yeah this change wasn't strictly necessary so we can revert it if it's causing issues.